folly/experimental/StringKeyedMap.h

```diff
@@ -191,7 +191,7 @@ class StringKeyedMap : private std::map<StringPiece, Value, Compare, Alloc> {
     Base::clear();
   }
 
-  void swap(StringKeyedMap& other) & {
+  void swap(StringKeyedMap& other) & noexcept(noexcept(Base::swap(other))) {
```

Comment:
Can this just be changed to `using Base::swap;`?

Reply:
Yep, looks like it. Just changed it. 👍